config/install/README.md

```diff
@@ -0,0 +1,87 @@
+# Install Kaudrant and Sail via OLM
+
+- Pre-Req is that OLM (operator lifecycle manager) is already installed
+
+- (optional dependencies)
+  - If you want to use `TLSPolicy` you should install the cert-manager operator. 
+  - AWS/Azure or GCP with DNS capabilities if you want to make use of `DNSPolicy`.
+  - Accessible Redis instance, if you want persistent storage for your rate limit counters.
+
+
+Install the Sail and Kuadrant Operators via OLM:
+
+
+> Note: By default this will install the "latest" or "main" of kuadrant. To change that, pick a release from the releases page in the kuadrant operator and change the image in the `config/deploy/olm/catalogsource.yaml` or if you are familiar with kustomize you could apply your own kustomization.
+
+```
+kubectl apply -k config/install/standard
+``` 
+
+3) verify kuadrant and sail operators are installed. Note this can take a while. You can also take a look at the subscription and installplan resource to help with debugging but the end state should be as below:
+
+```
+kubectl get deployments -n kuadrant-system
+
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+authorino-operator                      1/1     1            1           83m
```

Comment:
This add the output to the command to so it breaks it you could add it as comments or as a separate code block